kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaServiceConfiguration.java

```diff
@@ -74,6 +74,11 @@
     //
     // --- Kafka on Pulsar Broker configuration ---
     //
+    @FieldContext(
+            category = CATEGORY_KOP,
+            doc = "The number of threads used to respond to the response."
+    )
+    private int sendKafkaResponseThreads = Runtime.getRuntime().availableProcessors();
```

Comment:
It's better to use `numSendKafkaResponseThreads` as the name, like `numIOThreads`.

Reply:
OK，I'll fix it